model/Licensing/Properties/seeAlso.md

```diff
@@ -4,11 +4,27 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+A seeAlso contains a URL where the License or LicenseException can be found
+in use.
 
 ## Description
 
-TODO
+A seeAlso defines a cross-reference with a URL where the License or
+LicenseException can be found in use by one or a few projects.
```

Comment:
```suggestion
LicenseException can be found in use by one or more projects.
```

Reply:
I think "a few" is correct here. The general suggestion is that a few cross-references are appropriate, but doesn't need to list every project that uses this license.